diesel/Cargo.toml

```diff
@@ -17,6 +17,7 @@ byteorder = "0.3.*"
 quickcheck = { version = "0.2.25", optional = true }
 chrono = { version = "^0.2.17", optional = true }
 uuid = { version = "^0.2.0", optional = true, features = ["use_std"] }
+log = { version = "0.3", optional = true }
```

Comment:
I don't think we should use the `log` crate here. It looks like it only works with a global logger, which isn't a route I'd like to go down. Since everything would be the same log level anyway, I don't see much reason to take anything other than a `W: Write`.